NEWS.md

```diff
@@ -0,0 +1,40 @@
+# shiny.router (development version)
+
+- Changed the dots (`...`) argument in `router_ui()` to allow dynamically passed arguments
+- Fixed the issue where the 404 page was not working when a non-valid page is opened as the first page
+
+# shiny.router 0.3.0
+
+- Added a new API that is compatible with Rhino
+- `router_ui` and `router_server` replaced `make_router`
+- `router_ui` should be applied inside the UI function (not outside, like `make_router`) and thanks to that, can utilize `ns()` if used in a Rhino app
+- `router_server` adds all required server components (mostly `observeEvents`) to the server part of the application
+- Marked `make_router` as deprecated, it is not removed
```

Comment:
1. It suggests that introducing `router_ui` and `router_server` is something different than the new API. This has to be a single point. Don't add so many low-level technical details. It should include information that there is a new API compatible with Rhino (you can mention function names) and the old one is deprecated.